packages/@okta/vuepress-site/docs/guides/configure-amr-claims-mapping/main/index.md

```diff
@@ -0,0 +1,126 @@
+* An existing OpenID Connect Identity Provider (IdP) that’s able to send AMR claims to Okta. This can be another Okta org (org2org) or a third party IdP.
+* The IdP AMR Claims Mapping feature enabled for your org. From the left navigation pane in the Admin Console, go to **Settings** > **Features**, locate **Trust AMR claims from external IdPs**, and enable.
+
+---
+
+## Overview
+
+Authentication Method Reference (AMR) claims mapping allows an admin to configure their Okta org to accept AMR claims from OpenID Connect IdPs during SSO. Mapping AMR claims from third-party IdPs allows Okta to interpret the authentication context from an IdP. This helps eliminate duplicate factor challenges during user authentication.
+
+AMR claims provide important context to Okta during policy evaluation. For example, AMR claims give Okta a better understanding of which factors were used by the external IdP to verify the user's identity. This creates a more seamless and secure user experience, reduces friction, and boosts productivity.
+
+## AMR claims mapping flow
+
+<div class="three-quarter">
+
+![Flow diagram that displays the back and forth between the user, user agent, authorization server, and the Identity Provider](/img/auth/amr-claims-mapping-oidc.png)
+
+</div>
+
+1. A user attempts to sign in to an OpenID Connect app through the browser (user agent).
+2. The browser redirects the user to the Okta `/authorize` endpoint to authenticate.
+3. Okta redirects the user to the external Identity Provider.
+4. The Identity Provider authenticates the user.
+5. The Identity Provider redirects the user to Okta. The Identity Provider response contains the supported AMR claims, for example: `sms`, `mfa`, and `pwd`.
+
+    > **Note:** The AMR claims are stored in an Okta session and considered during policy evaluation.
+
+6. Okta redirects the user to the browser. The user isn't challenged for MFA if the policy requirements are met by the factors used by the Identity Provider.
+7. The browser sends a request to the Okta `/token` endpoint.
+8. Okta responds with the access token and the AMR claims in the ID token.
+
+## Configure the IdP for AMR claims
+
+Before you configure Okta to accept AMR claims, it’s important to first configure the IdP to send the claims correctly. Every IdP is different. Okta expects the IdP to pass the AMR claims in a specific way, depending on the supported federation protocol.
+
+### OpenID Connect Identity Provider
+
+At the OpenID Connect IdP, verify that the client app that you use for authenticating and authorizing users sends an `amr` array with the AMR claims in the OpenID Connect ID token (`id_token`).
+
+The `amr` property is a JSON array of strings that are identifiers for [authentication methods](https://www.rfc-editor.org/rfc/rfc8176.html) used in the authentication. <!-- Supported values include "pwd", "mfa", "otp", "kba", "sms", "swk", and "hwk". Get Venkat input on listing all supported values in the next update of this doc -->
```

Comment:
The commented-out content displays in the netlify link ... not sure if this will happen for regular builds? If you do want to add this content, suggest we include values in code font, rather than in quotes.

Reply:
it's an old netlify link - so that's before I commented it out to use for future updates. Thanks